access-checkout-react-native-sdk/README.md

```diff
@@ -11,14 +11,23 @@ It is designed to simplify the integration of the following functionality in you
 
 ## Documentation
 
-You can find the detailed documentation explaining how to integrate the SDK and use a session to take a payment starting with the [React Native section](https://developer.worldpay.com/docs/access-worldpay/checkout/react-native) of the [Access Worldpay documentation](https://developer.worldpay.com).
+You can find the detailed documentation explaining how to integrate the SDK and use a session to take a payment starting with the [React Native section](https://developer.worldpay.com/docs/access-worldpay/checkout/react-native) of the [Access Worldpay Checkout documentation](https://developer.worldpay.com).
 
 ## Compatibility
 
 - all `react-native` versions >= `0.61.5`
 - all `react` versions >= `16.9.0`
 - `Cocoapods` only for iOS dependencies
 
+## SAQ-A Compliance
+
+AccessCheckoutTextInput is the name of the new component created to provide SAQ-A compliance when using our React Native SDK.
```

Comment:
I think it may be best to invert the sentence to convey first what is the goal that we are trying achieve (i.e. SAQ-A compliance) because it gives a bit of context first.

_To support SAQ-A compliance when using our React Native SDK, we have created a new component called `AccessCheckoutTextInput`._ 

Plus add a mention about card details in the next sentence to explain why this is functionally important:

It has been designed so that it does not expose any methods or properties to retrieve the text entered by the end user to ensure our merchants applications _do not have direct access to card details_ and are SAQ-A compliant.